modules/upi-provisioning-new-hosts-in-a-upi-cluster.adoc

```diff
@@ -20,45 +20,156 @@ To provision bare-metal hosts to the cluster by using the BMO, you must set the
 
 .Procedure
 
-. Create the `Secret` CR and the `BareMetalHost` CR.
-
-.. Save the following YAML in the `bmh.yaml` file:
+. Create a configuration file for the bare-metal node. Depending on whether you are using a static configuration or a DHCP server, use one of the following example `bmh.yaml` files, replacing values in the YAML to match your environment:
++
+.Static configuration `bmh.yaml`
 +
 [source,yaml]
 ----
 ---
 apiVersion: v1
 kind: Secret
 metadata:
-  name: worker1-bmc
+  name: openshift-worker-<num>-network-config-secret <1>
+  namespace: openshift-machine-api
+type: Opaque
+stringData:
+  nmstate: | <2>
+    interfaces: <3>
+    - name: <nic1_name> <4>
+      type: ethernet
+      state: up
+      ipv4:
+        address:
+        - ip: <ip_address> <5>
+          prefix-length: 24
+        enabled: true
+    dns-resolver:
+      config:
+        server:
+        - <dns_ip_address> <6>
+    routes:
+      config:
+      - destination: 0.0.0.0/0
+        next-hop-address: <next_hop_ip_address> <7>
+        next-hop-interface: <next_hop_nic1_name> <8>
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: openshift-worker-<num>-bmc-secret
   namespace: openshift-machine-api
 type: Opaque
 data:
-  username: <base64_of_uid>
+  username: <base64_of_uid> <9>
   password: <base64_of_pwd>
 ---
 apiVersion: metal3.io/v1alpha1
 kind: BareMetalHost
 metadata:
-  name: worker1
+  name: <openshift-worker-<num>
   namespace: openshift-machine-api
 spec:
+  online: true
+  bootMACAddress: <nic1_mac_address> <10>
   bmc:
-    address: <protocol>://<bmc_url> <1>
-    credentialsName: "worker1-bmc"
-  bootMACAddress: <nic1_mac_address>
-  externallyProvisioned: false <2>
+    address: <protocol>://<bmc_url> <11>
+    credentialsName: openshift-worker-<num>-bmc-secret
+    disableCertificateVerification: false
   customDeploy:
     method: install_coreos
+  userData:
+    name: worker-user-data-managed
+    namespace: openshift-machine-api
+  rootDeviceHints:
+    deviceName: <root_device_hint> <12>
+  preprovisioningNetworkDataName: openshift-worker-<num>-network-config-secret
```

Comment:
This is the end of the file or is # ... needed? See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#yaml-formatting.

Reply:
It's EOF